.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,59 @@
+name: Publish release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version number to tag and release'
+        required: true
+        type: string
+      prerelease:
+        description: 'Release as pre-release'
+        required: false
+        type: boolean
+        default: false
+
+jobs:
+  release-npm:
+    runs-on: ubuntu-latest
+    environment: main
+    permissions:
+      contents: write
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/*'
+          cache: 'npm'
+          cache-dependency-path: package.json
+          check-latest: true
+
+      - name: Install
+        run: |
+          npm install --ignore-scripts
+
+      - name: Bump version
+        id: bump
+        run: |
+          new_version=$(npm version ${{ inputs.version }} --no-git-tag-version | tail -n 1)
+          echo "version=$new_version" >> $GITHUB_OUTPUT
```

Comment:
we need to update this version too:
https://github.com/fastify/fastify/blob/main/fastify.js#L3